static/templates/disbursements.hbs

```diff
@@ -45,6 +49,20 @@
 </div>
 
 <div class="panel__row">
+  <h4 class="panel__title">Filing information</h4>
+  <table>
+    <tr>
+      <td class="panel__term">Filing date</td>
+      <td>{{{ datetime receipt_date }}}</td>
+    </tr>
+    <tr>
+      <td class="panel__term">View report page</td>
+      <td><a href="{{{ pdf_url }}}" target="_blank">View on fec.gov</a></td>
```

Comment:
Well, I am blind. Thanks for spotting that! 

A thought on the language, what if it says--  "View report page" instead of "View original image"  Does @emileighoutlaw or @noahmanger have strong feelings on this micro copy?